pandas/core/index.py

```diff
@@ -256,6 +257,11 @@ def __array_wrap__(self, result, context=None):
         """
         return self._shallow_copy(result)
 
+    def reshape(self, shape, **kwargs):
+        """ reshape the underlying data; this is purely for compat
+        and has no effect on a 1-d Index type """
+        return self
```

Comment:
This seems like a bad idea. `reshape` should raise a ValueError if the desired shape is not 1D.